requirements.txt

```diff
@@ -0,0 +1,24 @@
+astropy
+hypothesis==6.29.*
+matplotlib
+more-itertools
+numpy>1.21.*
+openpyxl
+pytest
+pytz
+tabulate
+scipy
+coverage
+requests
+pandas
+aiorun
+omegaconf
+python-dotenv
+fastapi
+strawberry-graphql[fastapi]
+uvicorn[standard]
+gql[all]
+backoff
+lucupy
+pytest-asyncio
+aiorun
```

Comment:
The docs should probably refer to this file rather than `environment.yml` given that the requirements are here.